src/scripts/main.js

```diff
@@ -1,3 +1,166 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+let counterClick = 0;
+let prevSorted = '';
+
+thead.addEventListener('click', (e) => {
+  const index = e.target.cellIndex;
+  const data = [...tbody.children];
+
+  if (prevSorted === e.target) {
+    counterClick++;
+  } else {
+    counterClick = 0;
+  }
+
+  data.sort((a, b) => {
+    const contentA = counterClick % 2 === 0
+      ? a.cells[index].textContent
+      : b.cells[index].textContent;
+    const contentB = counterClick % 2 === 0
+      ? b.cells[index].textContent
+      : a.cells[index].textContent;
+
+    switch (index) {
+      case 0 : return contentA.localeCompare(contentB);
+
+      case 1 : return contentA.localeCompare(contentB);
+
+      case 2 : return contentA.localeCompare(contentB);
+
+      case 3 : return contentA - contentB;
+
+      case 4 : return parseInt(contentA.slice(1)) - parseInt(contentB.slice(1));
+    }
+  });
+  tbody.append(...data);
+
+  prevSorted = e.target;
+});
+
+tbody.addEventListener('mouseover', (el) => {
+  const row = el.target.closest('tr');
```

Comment:
Looks good, but according to the [description](https://github.com/mate-academy/js_employees_table_DOM#when-user-clicks-on-a-row-it-should-become-selected) it should be active on click:
![image](https://user-images.githubusercontent.com/86923089/219948750-db82b482-bfc3-41dd-9083-65412babe7db.png)